app/assets/stylesheets/_state-file.scss

```diff
@@ -672,6 +672,24 @@
       display: none
     }
   }
+
+  &.us-state--az {
```

Comment:
could this go in the `@each $state_key, $state-color in $state-colors {` block elsewhere in this file, so that we don't have to manually define a new style for the intercom button for each state?

Reply:
Done - but I also had to add a default color for when no state is selected (Like in login-options)